keadm/cmd/keadm/app/cmd/edge/image.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 func request(opt *common.JoinOptions, step *common.Step) error {
-	imageSet := image.EdgeSet(opt.ImageRepository, opt.KubeEdgeVersion)
+	imageSet := image.EdgeSet(opt, opt.KubeEdgeVersion)
```

Comment:
Is it better to use opt as the only param, like EdgeSet(opt)

Reply:
yeah, it's a good suggestion, and i have made it